cmd/pdcli/oncall/cmd.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright © 2021 Red Hat, Inc
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oncall
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/olekukonko/tablewriter"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/constants"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "oncall",
+	Short: "oncall to the PagerDuty CLI",
+	Long:  "Running the pdcli oncall command will display the current primary and secondary oncall SRE",
+	Args:  cobra.NoArgs,
+	RunE:  OnCall,
+}
+
+//TimeConversion converts timestamp into time and date
+func TimeConversion(s string) string {
```

Comment:
This can be a local function instead.

Reply:
@MitaliBhalla it doesn't look like  this function need to be exported outside oncall module
```suggestion
func timeConversion(s string) string {
```
note: references to this function will need to be updated too